packaging/nntrainer.spec

```diff
@@ -254,6 +268,8 @@ cp -r result %{buildroot}%{_datadir}/nntrainer/unittest/
 %postun -p /sbin/ldconfig
 
 %files
```

Comment:
remove this? as nntrainer package depends on nntrainer-core, these files will be installed by core.

Reply:
This should be empty package. It's only for the resolve dependency.

Reply:
I am not so sure about this.
Please once check - https://wiki.tizen.org/Security/Application_installation_and_Manifest. It says that "Each RPM package in MSM must have a manifest file".

Reply:
I think this is not actually installed. The packages which has dependency with it and has manifest file are installed.

Reply:
If a package does NOT have a binary or executable (.so, .out, .sh, and so on), it has no execution incurred by the package, so MSM has nothing to register for the package. So, if the package has no "execution files", you don't need to care MSM. That's why we do not add manifest file to packages without .so, .out, .sh files.